src/ui/public/kbn_share/kbn_share.less

```diff
@@ -0,0 +1,42 @@
+.sharing-ui-menu__List {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.sharing-ui-menu__ListWrapper,
+.sharing-ui-menu__SharePanel {
```

Comment:
So, that's what confuses me about BEM. I like doing, for example, `.sum-share-panel` where `.sum` means Sharing UI Menu. How do I differentiate between "this is the namespace" and "this is the component"? I reserved hyphen-case for states (`.sharing-ui-menu__ListWrapper-is-open`)